### ISSUE TITLE
Normalisation requires full plates on non-normalized tsvs.

### ISSUE THREAD

**Issue by dirmeier** [cbg-ethz/rnaiutilities#23]
So far, when we norm data, we assume, the complete plate is represented.
However, when the user only queried for some genes or so, this wouldn't work, for example:

```bash
 rnai-query query --gene pik3ca -- sample 10 FILE
```

This would standardize all effects, with the result that basically the signals are entirely lost. Normalisation always needs to be done against the complete plate context. So the norm has to be prior to writing tsvs.

**Comment by MartinFXP** [comment]
I would suggest something like

```
rnai-query query --gene pik3ca --sample 10 --norm TRUE FILE
```

to let the user decide if he wants to have normalized or raw data.

**Comment by dirmeier** [comment]
Yeah, I thought of that. But it doesnt specify *how* you normalize, e.g. at some point you want to correct for spatial effects, fit a spline to correct for cell numbers and standardize. Then you would need to:

```bash
   rnai-query query --gene pik3ca --sample 10 --norm bscore,spline,zscore FILE
```

However, I'd like to avoid having to specify dozens of command line flags. `rnai-query` has already 3 subcommends and like 10 (?) flags.

**Comment by dirmeier** [comment]
[This release](https://github.com/cbg-ethz/rnaiutilities/releases/tag/v0.1.3.post1) introduces simple plate standardization. I'll guess we need to add some more methods, too. So I'll leave this open.
